### ISSUE TITLE
[Badge] Review accessibility

### ISSUE THREAD

**Issue by diogofscmariano** [lumada-design/hv-uikit-react#1234]
**Description**
We want to make sure the badge can be configured and include the aria label attributes properly.

- No aria live will be added because we assume the app will use another component to announce any app related notifications
- Add aria label, which reflects the count the component has
- All its child elements should have the aria-presentation role.

**Acceptance Criteria**
- Add accessibility properties properly
- Add sample which shows the added accessibility attributes
